plugins/alerting/pkg/alerting/endpoints/v1/server.go

```diff
@@ -235,6 +240,36 @@ func (e *EndpointServerComponent) TestAlertEndpoint(ctx context.Context, req *al
 		return nil, err
 	}
 	go func() { // create, trigger, delete
+		start := time.Now()
```

Comment:
Perhaps this function should just block until ctx is canceled to avoid arbitrary delays

Reply:
the decision was made for the api not to block on a long "send and forget" pipeline, the errors don't return any actionable information for the end user anyways, and the end user may not receive the message until minutes after the end of the non-blocking call anyway

Reply:
I removed some of the debug logs, and timings

Reply:
this API is obsolete, it was moved and reworked to the notification server : it was kept in endpoints server to satisfy the existing API contract but forced alerting to do an ugly dependency injection